src/db/collections/mod.rs

```diff
@@ -23,3 +23,53 @@ pub use self::{
     },
     treasury::TreasuryResult,
 };
+use crate::types::stardust::block::{
+    AliasOutput, BasicOutput, FoundryOutput, MilestonePayload, NftOutput, TaggedDataPayload, TransactionPayload,
+};
+
+/// Helper to specify a kind for an output type.
+pub trait OutputKind {
+    /// Gets the output kind.
+    fn kind() -> Option<&'static str> {
```

Comment:
Why do we need the `Option` here (and for `PayloadKind`)? I think we can just leave it out.

Reply:
You think I would have designed it this way if I didn't have a reason? 😆 

It is so that we can _not_ specify a kind.

Reply:
Ah for the overall call?

Reply:
Gottcha